werf-giterminism.yaml

```diff
@@ -1,7 +1,7 @@
 giterminismConfigVersion: 1
 config:
   goTemplateRendering:	# The rules for the Go-template functions
-    allowEnvVariables: [ /CI_.+/, /REPO_MCM_.+/, SOURCE_REPO, GOPROXY, WERF_DISABLE_META_TAGS, CLOUD_PROVIDERS_SOURCE_REP, OBSERVABILITY_SOURCE_REPO ]
+    allowEnvVariables: [ /CI_.+/, /REPO_MCM_.+/, SOURCE_REPO, GOPROXY, WERF_DISABLE_META_TAGS, CLOUD_PROVIDERS_SOURCE_REPO, OBSERVABILITY_SOURCE_REPO ]
```

Comment:
Error: template: werfConfig:61:47: executing "werfConfig" at <env "CLOUD_PROVIDERS_SOURCE_REPO">: error calling env: the configuration with potential external dependency found in the werf config: env name "CLOUD_PROVIDERS_SOURCE_REPO" not allowed by giterminism